CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 1.3.1 - 2023-02-01
+### Added
+- Allows userinfo to be queried by reference id
```

Comment:
put this under "unreleased" it will automatically create the date (check the previous closed PR)